src/utils/env.ts

```diff
@@ -0,0 +1,20 @@
+export const isLocalhostDomain = (
+    hostname = window.location.hostname
+): boolean => {
+    return hostname === 'localhost';
```

Comment:
Maybe some unnecessary returns here, `=> hostname === 'localhost'` feels clear enough to me. Return types can also be inferred without specifying.